v22.2/use-cloud-storage-for-bulk-operations.md

```diff
@@ -138,6 +138,115 @@ You [can associate an EC2 instance with an IAM role](https://docs.aws.amazon.com
 aws ec2 associate-iam-instance-profile --iam-instance-profile Name={example profile} --region={us-east-2} --instance-id {instance example}
 ~~~
 
+### Assume role authentication
+
+{{site.data.alerts.callout_info}}
+CockroachDB supports assume role authentication on clusters running v22.2. Authenticating to cloud storage with `ASSUME_ROLE` on clusters running versions v22.1 and earlier, or mixed versions, is not supported and will result in failed bulk operations.
+{{site.data.alerts.end}}
+
+{% include_cached new-in.html version="v22.2" %} To limit the control over Amazon S3 buckets, you can create IAM roles for users to assume. IAM roles do not have an association to a particular user. The role contains permissions that define the operations a user (or [Principal](https://docs.aws.amazon.com/IAM/latest/UserGuide/intro-structure.html#intro-structure-principal)) can complete. An IAM user can then assume a role to undertake a CockroachDB backup, restore, import, etc. As a result, the IAM user only has access to the assigned role, rather than having unlimited access to an S3 bucket. 
```

Comment:
Nit: `limit the control` might be more readily understood as `To more tightly control access to your Amazon S3 buckets` -- we aren't really changing _control_ of the buckets so much as more finely-controlling _access_ to them. Very nit, feel free to disregard.